src/RosettaPy/rosetta.py

```
@@ -92,33 +92,45 @@
                 self.bin = self.run_node.rosetta_bin
             else:
-                # local direct runs
+                # otherwise (MpiNode and Native), search for local installations
                 self.bin = RosettaFinder().find_binary(self.bin)
 
+        # explicitly disable MPI for Native node
+        if isinstance(self.run_node, Native):
+            self.use_mpi = False
+            # warnings about the disabled MPI mode for MPI-supported binaries.
+            if self.bin.mode == "mpi":
+                warnings.warn(UserWarning("The binary supports MPI mode, yet the job is not configured to use MPI."))
```

Ruff

No explicit `stacklevel` keyword argument found

(B028)